src/Calendar/Calendar.js

```diff
@@ -110,7 +111,10 @@ class Calendar extends Component {
 
     changeMonth = (month) => {
         let date = this.state.currentDateDisplayed;
-        let months = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'];
+        const { monthJan, monthFeb, monthMar, monthApr, monthMay, monthJun, monthJul,
+            monthAug, monthSep, monthOct, monthNov, monthDec } = this.props.localizedText;
+        let months = [monthJan, monthFeb, monthMar, monthApr, monthMay, monthJun, monthJul,
+            monthAug, monthSep, monthOct, monthNov, monthDec];
```

Comment:
Couple things here:

1. For readability, I'd put each value from `localizedText` on its own line such as:
   ```
   const {
      monthJan,
      monthFeb,
      monthMar,
      ...
   } = this.props.localizedText;
   ```
2. Since this work is done in multiple places, let's make a function that returns the array of localized month names.  That function can do the extraction of values from the `localizedText` object and the array construction.